app/views/shared/views/ems_common/_form.html.haml

```diff
@@ -28,7 +28,7 @@
         - else
           = h(@edit[:ems_types][@edit[:new][:emstype]])
     = render(:partial => "form_fields", :locals => {:url => url})
-    - if @edit[:new][:emstype] && @edit[:new][:emstype] == 'rhevm'
+    - if @edit[:new][:emstype] && %w(rhevm openstack_infra).include?(@edit[:new][:emstype])
```

Comment:
@chessbyte also the model method already exist, for some reason, those are used only in controllers. 

It's called:
ems.supports_port?

So we will need to do a bigger cleanup, I think best option will be to clean the angular form and exposing model methods there.